src/components/Charts/ModelChart.js

```diff
@@ -45,7 +46,7 @@ const condensedFormatIntervention = (intervention, optCase) =>
 const ModelChart = ({
   height,
   condensed,
-  countyName,
+  isCounty,
```

Comment:
the Projections model as a concept of county, and we should probably just add a isCounty method on the projections model instead of passing this everywhere

Reply:
so would just be interventions.isCounty()

Reply:
Ooh, nice!  Done.  I made it a field instead of a method to be consistent with the existing `hasProjections` field I noticed.  We could make them both methods if we want to avoid storing redundant state (or we could make them properties with getters!). 🤷